exchange/adapter_util.go

```diff
@@ -63,6 +68,23 @@ func buildBidders(infos config.BidderInfos, builders map[openrtb_ext.BidderName]
 	return bidders, errs
 }
 
+func SetAliasBuilder(info config.BidderInfo, builders map[openrtb_ext.BidderName]adapters.Builder, bidderName openrtb_ext.BidderName) error {
```

Comment:
scope of `SetAliasBuilder` is within `adapter_util` package. should make this method private

Reply:
made the method private in [484d3a0](https://github.com/prebid/prebid-server/pull/3016/commits/484d3a0f442a5bff702da2860ae8dd1338737285)